neo4j/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # CHANGELOG - neo4j
 
-## 2.0.2 / 2023-04-28
-* [Fixed] Remove the use_latest_spec option from the config file. See [#1835](https://github.com/DataDog/integrations-extras/pull/1835).
+## 3.0.0 / 2023-05-12
+* [Changed] Update Neo4j Integration to support Neo4j 5. 
```

Comment:
```suggestion
* [Added] Update Neo4j Integration to support Neo4j 5. 
```

Non blocking nit. Since we're no longer changing the metrics in favor or Neo4j 5 versions, I think we can make this a `[Added]` change since this will be an addition

Reply:
Changed. Thanks.